src/main/webapp/app/config/constants.tsx

```diff
@@ -24,6 +24,8 @@ export const DISABLE_BANNER_OPT = 'disablebanner';
 export const DEV_URL = 'http://localhost:9095';
 
 export const RECAPTCHA_SITE_KEY = '6LcxRsMZAAAAAFYpXX6KAc9ASGSf8IptsIKehJby';
+export const RECAPTCHA_ENTERPRISE_SITE_KEY =
```

Comment:
Do we need both?

Reply:
I believe I deleted the unnecessary key. Will double check